repos/system_upgrade/common/actors/kernelcmdlineconfig/libraries/kernelcmdlineconfig.py

```diff
@@ -109,10 +109,47 @@ def _extract_grubby_value(record):
     return matches.group(2)
 
 
+def report_multple_entries_for_default_kernel():
+    if use_cmdline_file():
+        report_hint = (
+            'After the system has been rebooted into the new version of RHEL, you'
+            ' should set the correct kernel cmdline arguments in /etc/kernel/cmdline '
+            ' and reboot the system to apply them.'
```

Comment:
```suggestion
            'After the system has been rebooted into the new version of RHEL, you'
            ' should check that configured default kernel cmdline arguments in /etc/kernel/cmdline '
            ' are correct. In case that different arguments are expected, update the file as needed.'
```